examples/gallery/maps/roads.py

```diff
@@ -0,0 +1,42 @@
+"""
+Roads
+-----
+The :meth:`pygmt.Figure.plot` method allows to plot geographical data which is
+stored in a :class:`geopandas.GeoDataFrame` object.
```

Comment:
I think the description could be made a bit longer to describe the data loading and plotting/styling process. Feel free to edit before committing.

```suggestion
The :meth:`pygmt.Figure.plot` method allows us to plot geographical data such
as lines which are stored in a :class:`geopandas.GeoDataFrame` object. Use
:meth:`geopandas.read_file` to load data from any supported OGR format such as
a shapefile (.shp), GeoJSON (.geojson), geopackage (.gpkg), etc. Then, pass the
:class:`geopandas.GeoDataFrame` as an argument to the `data` parameter in
:meth:`pygmt.Figure.plot`, and style the geometry using the `pen` parameter.
```